### PR TITLE
Added tooltip options constructor, barpercentage in x-axis

### PR DESCRIPTION
**Pull request by elayabharath**
https://www.chartjs.org/docs/latest/charts/bar.html#dataset-configuration
barPercentage - Percent (0-1) of the available width each bar should be within the category width.

### CODE REVIEW

File: lib/chartjs.dart [hunk 1]
```diff
@@ -863,6 +863,7 @@ abstract class ChartTooltipOptions {
       dynamic /*String|CanvasGradient|CanvasPattern|List<String>*/ v);
   external num get borderWidth;
   external set borderWidth(num v);
+  external factory ChartTooltipOptions();
```
Comment (marianamarasoiu) [inline lib/chartjs.dart]: I know it's a bit tedious, but could you also add all the fields in the constructor as optional named arguments? thanks!

**Review by marianamarasoiu — APPROVED** [review]
thanks @elayabharath!

File: lib/chartjs.dart (hunk 1 above)
Comment (elayabharath) [reply lib/chartjs.dart]: Sorry yes, added it

**Review by marianamarasoiu — APPROVED** [review]
great, thanks @elayabharath !

**Pull request merged**